docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md

```diff
@@ -106,10 +106,42 @@ etc/ bin/ share/ scripts/
 Since C++ templates are heavily used by **Nebula Graph** and its third party dependencies, especially Folly, fbthrift and boost, the building is very time-consuming.
 For your reference, it is expected to take about 35 minutes in CPU time(less than 4 minutes with `-j16`), given an Intel E5-2697 v3 processor and unit tests are disabled.
 
-Until now, you might already have built **Nebula Graph** successfully. If so or not, we also provide ways to tweak the building process.
+### Starting **Nebula Graph** Services
+
+After compiling and installing **Nebula Graph**, you can start **Nebula Graph** services. By default, **Nebula Graph** is installed in the `/home/username/nebula/build/install` directory, in which the *username* must be replaced by your username.
+
+1. Change your current directory to the directory where **Nebula Graph** is installed.
+
+```bash
+$ cd /home/username/nebula/build/install
+```
+
+2. Rename the configuration files for **Nebula Graph** services.
```

Comment:
I think this means copy and remane

Reply:
Exactly, we can update later as it does not have any impact on any further operation